evap/rewards/views.py

```diff
@@ -26,24 +27,33 @@
 from evap.staff.views import semester_view
 
 
+def redeem_reward_points(request):
+    redemptions = {}
+    try:
+        for key, value in request.POST.items():
+            if key.startswith("points-"):
```

Comment:
Out of curiosity: Is there a reason why we extract this information manually? Could we instead do it using standard django tools, e.g. using formset logic? I'd then make a new issue for that.

Reply:
Yeah, probably possible. Maybe the way how we transmit the mapping of event-id and points need to be changed as well